ecommerce/src/main/resources/application.yml

```
@@ -124,3 +144,34 @@ management:
+      dataPlatform:
+        registerHealthIndicator: true     # Actuator에 상태 노출
+        slidingWindowType: COUNT_BASED    # 카운트 기반 슬라이딩 윈도우
+        slidingWindowSize: 10             # 최근 10개 요청 기준
+        minimumNumberOfCalls: 5           # 최소 5개 요청 후 판단
+        failureRateThreshold: 50          # 실패율 50% 이상 시 Open
+        slowCallRateThreshold: 80         # 느린 호출 80% 이상 시 Open
+        slowCallDurationThreshold: 3s     # 3초 이상이면 느린 호출
+        waitDurationInOpenState: 30s      # Open 상태 유지 시간
+        permittedNumberOfCallsInHalfOpenState: 3  # Half-Open에서 허용 호출 수
+        automaticTransitionFromOpenToHalfOpenEnabled: true
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)